src/Education/XamlGenerator/RuleHelpXamlTranslator.cs

```diff
@@ -307,6 +309,11 @@ private void ProcessElement()
 
                         break;
 
+                    case "span":
+                        WriteInlineElementStart("Span");
```

Comment:
Does it display correctly? Also add a simple test for this, maybe even parse to make sure

Reply:
Tests are in place already, that's how we have found the issue. And I have manually checked for failing rules and they were displayed correctly.

Reply:
Those are smoke tests. We also have `RuleHelpXamlTranslatorTests`. It's not a complete set of test cases, but IMO it's good to add tests for new stuff we add, especially if it's small things